index.js

```diff
@@ -81,19 +92,23 @@ let Rosnodejs = {
    * @return {Promise} resolved when connection to master is established
    */
   initNode(nodeName, options) {
+    options = options || {};
+    if (options.anonymous) {
+      nodeName = _anonymizeNodeName(nodeName);
+      console.log('anonymous node name ' + nodeName);
```

Comment:
It's not necessarily bad to have it in there, as the pid and time are not obvious to the user when starting the node.